src/content/blog/2024-06-20-simple-open-payments-guide.md

```diff
@@ -0,0 +1,102 @@
+
+While some fees, such as currency conversion and bank transaction fees, may still apply, Open Payments aims to eliminate unnecessary middleman fees, reducing overall transaction costs. However, it’s important to recognize that not all intermediaries are unnecessary. For example, currency conversion services are essential for transactions involving different currencies. Suppose your account provider only deals in Euros and mine only in Dollars. In that case, we might rely on an intermediary that handles both currencies to facilitate the transaction effectively.
+
+Another scenario arises when the sender's and recipient's account providers are not directly connected by a shared settlement layer. In such cases, as long as there are one or more intermediaries that share a common settlement system with each institution, settlement can still occur via these intermediaries. For instance, if my bank deals in Dollars and cannot settle directly with your cryptocurrency wallet, there might be a digital wallet intermediary capable of settling between fiat and cryptocurrency. Thus, a shared settlement layer exists indirectly through intermediaries. The goal of the Interledger Protocol is to navigate the network of financial institutions and settlement systems to find the most efficient and cost-effective settlement paths. By leveraging these interconnected networks, Interledger aims to minimize costs and enhance reliability.
+
+## What's Happening in the Open Payments Space Today?
+
+<!-- TODO:  -->
+
+- INCOMPLETE: Is there anyone in the wild using OP now?
+
+Currently, the adoption of the Open Payments standard is still in progress. Some innovative institutions and services have begun to integrate this standard, but widespread use is still developing.
+
+[GateHub](https://gatehub.net/) is a digital wallet provider working with the Open Payments standard globally. They are able to facilitate some cross-currency transactions, although regulatory limitations may apply depending on the user's country of residence. The [Fynbos Wallet](https://wallet.fynbos.app) is another example. They are operational in America, Europe, and South Africa. However, transactions are currently limited to other Fynbos users within the same region due to regulatory and technical constraints. There are plans underway to create payment channels between Fybos and GateHub users in Europe.
```

Comment:
"There are plans underway to create payment channels between Fybos and GateHub users in Europe."

I think not just Europe but all Gatehub to all Fynbos users. But they may start with Europe.

Reply:
This is how Adrian phrased it to me. Since this is an OP intro I'm hesitant to sidetrack too much on making this a progress report and map of future developments.

We could try, "There are plans underway to create payment channels between Fybos and GateHub users." and just leave it there, or, "Plans are underway to create payment channels between Fynbos and GateHub users, beginning with Europe."